### DISCUSSION TITLE
[Q&A] Setting up

### DISCUSSION THREAD

**Discussion by forklift-operator** [QIN2DIM/hcaptcha-challenger#782] [General]
I can't find the src module with the main.py, can you help me?

**Comment by QIN2DIM** [comment]
wiki is outdated :< I am writing a new document

**Comment by QIN2DIM** [comment]
I find writing documents really time-consuming.

You can tell me what you need directly, and I will design the interface for you, because this project can't handle any token generator directly (like discord GM) , and a lot of adaptation code is needed in the middle.

But if what you need is the following simple "one step" interface, I can open a new branch to do a similar interface (because then I don't write the document you know how to use it :>)

https://git.anti-captcha.com/sup/anticaptcha-python

**Comment by duomaomao319** [comment]
新版本如何跑起来？Readme的示例out of date了

**Comment by QIN2DIM** [comment]
@duomaomao319 看 examples
